web_programming/instagram_video.py

```diff
@@ -0,0 +1,17 @@
+import json
+from datetime import datetime
+
+import requests
+
+if __name__ == "__main__":
+    link = input("Enter Video/IGTV url: ").strip()
+    url = f"https://downloadgram.net/wp-json/wppress/video-downloader/video?url={link}"
```

Comment:
```suggestion
def download_video(url: str) -> bytes:
    url = f"https://downloadgram.net/wp-json/wppress/video-downloader/video?url={url}"
```

Reply:
My sense is that it is too confusing for the readers if we reuse `url` as both a function parameter and a local variable.